test/gscraper/accounts_test.exs

```diff
@@ -0,0 +1,21 @@
+defmodule Gscraper.AccountsTest do
+  use Gscraper.DataCase
+
+  alias Gscraper.Accounts
+
+  describe "users" do
+    alias Gscraper.Accounts.User
+
+    test "create_user/1 with valid data creates a user" do
+      valid_params = params_for(:user, password: "P@ssw0rd")
+      assert {:ok, %User{} = user} = Accounts.create_user(valid_params)
```

Comment:
Let's add new line before this one to separate the setup and assertion section

Reply:
also fixed in 7c36f2d.